modules/loader/src/main/java/com/opengamma/strata/loader/csv/CsvLoaderUtils.java

```diff
@@ -228,22 +229,34 @@ public static Pair<YearMonth, EtdVariant> parseEtdVariant(CsvRow row, EtdType ty
         if (week == 0) {
           return Pair.of(yearMonth, EtdVariant.ofDaily(day));
         } else {
-          throw new ParseFailureException(
-              "Unable to parse ETD variant, date columns conflict, must not  set both expiry day and expiry week");
+          throw new ParseFailureException(FailureReason.INVALID,
+              "Unable to parse ETD variant, date columns conflict, must not set both expiry day and expiry week");
         }
       }
     } else {
       if (day == 0) {
-        throw new ParseFailureException("Unable to parse ETD variant for Flex '{value}', must set expiry day", type);
+        throw new ParseFailureException(
```

Comment:
These error messages are consistent at present and we need to not lose information here. Plus, error messages are not sentences - they do not have full stops.

Reply:
Have removed the full stops on all of these error messages as discussed.